OSDOCS-16941: DITA abstracts — OLM catalog/config topics (#115366)
openshift/openshift-docs: modules/olm-rh-catalogs.adoc

[id="olm-rh-catalogs_{context}"]
= About Red Hat-provided Operator catalogs

[role="_abstract"]

> Red Hat distributes default Operator catalogs as index images installed in the `{global_ns}` namespace on {product-title} clusters. These catalogs make Red Hat, certified, and community Operators available cluster-wide in the software catalog.

Reviewer: ```suggestion
The Red Hat-provided catalog sources are installed by default in the `{global_ns}` namespace, which makes the catalogs available cluster-wide in all namespaces.
```
